### ISSUE TITLE
`serde` attributes on types in conductor API are misused

### ISSUE THREAD

**Issue by Br1ght0ne** [holochain/holochain#420]
[`holochain::conductor::api`](https://github.com/holochain/holochain/blob/develop/crates/holochain/src/conductor/api/api_external/admin_interface.rs) has types that use `#[serde(rename = "snake-case")]` attributes.

https://github.com/holochain/holochain/blob/45dd3f827caea18f41f77486ca2c37149a18b4ca/crates/holochain/src/conductor/api/api_external/admin_interface.rs#L225-L226

https://github.com/holochain/holochain/blob/45dd3f827caea18f41f77486ca2c37149a18b4ca/crates/holochain/src/conductor/api/api_external/admin_interface.rs#L267-L268

According to [Serde docs on container attributes](https://serde.rs/container-attrs.html), `#[serde(rename = "...")]` should specify a new name for the container directly and not a conversion scheme, e.g. this is valid:

```rust
#[serde(rename = "my_struct")]
struct MyStruct {}
```

But this is **not**:

```rust
#[serde(rename = "snake_case")]
struct MyStruct {}
```

[Rust Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=8a640ec6d47a72621fa67477e98b1f3b) that shows the expected behavior.

---

cc @peeech

**Comment by zdb999** [comment]
I believe that #423 fixed this, so this issue can probably be closed.

**Comment by Br1ght0ne** [comment]
Closed in #423, thanks @zdb999!
